.github/workflows/main.yml

```diff
@@ -0,0 +1,128 @@
+name: Main
+
+on:
+  push:
+    branches:
+      - gh-actions
+#      - master
```

Comment:
What is the `gh-actions` branch used for? Shouldn't this workflow run on commits to `master`, or am I reading this wrong?

Reply:
Yep, it's my branch name, I overrode the branch to be able to test.

I fixed that.